### PR TITLE
JW-1307: make create-builder-template work on Codilime's CI

### PR DESCRIPTION
**Pull request by sodar**
- Moved original roles/vars from previous repository

    ```
    Author: Michał Bilski <michal.bilski@codilime.com>
    Author: Michał Kostrzewa <michal.kostrzewa@codilime.com>
    Author: Wojciech Urbański <wojciech.urbanski@codilime.com>
    ```

- Moved `create-builder-template` pipeline script to Jenkinsfile
- Fix some deployment issues

### CODE REVIEW

**Review by jacek-izykowski — DISMISSED** [review]

**Comment by m-kostrzewa** [comment]
@sodar which changes are file moves and which are actual changes to code?

File: jenkinsfiles/Jenkinsfile.create-builder-template [hunk 1]
```diff
@@ -0,0 +1,49 @@
+withCredentials([usernamePassword(credentialsId: 'e9f31c41-6d0e-4950-b6d7-1aa7080468aa', passwordVariable: 'VC_PASSWORD', usernameVariable: 'VC_USERNAME')]) {
```
Comment (wurbanski) [inline jenkinsfiles/Jenkinsfile.create-builder-template]: Consider reworking this to follow *declarative pipeline* style from `Jenkinsfile.deploy-builder` in the old repository.
Please. Strongly consider.

**Review by wurbanski — CHANGES_REQUESTED** [review]
Please show us what changed :).

**Comment by sodar** [comment]
@wurbanski @m-kostrzewa Sorry about the mess with the commits. I have rebased the changes, so that all moved/copied code is in one commit, i.e. the earliest commit in this PR.

I need to test the rest of the changes and refactor Jenkinsfile for create-builder-template so I mark this PR as WIP.

**Comment by sodar** [comment]
PR is ready for review. Commit with hash `999986...` contains my changes.

**Review by wurbanski — APPROVED** [review]
LGTM

**Review by m-kostrzewa — APPROVED** [review]

**Pull request merged**